weldx/asdf/schemas/weldx.bam.de/weldx/time/time-1.0.0.yaml

```
@@ -0,0 +1,59 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://weldx.bam.de/schemas/weldx/time/time-1.0.0"
+tag: "tag:weldx.bam.de:weldx/time/time-1.0.0"
+
+title: |
+  Represents a single or multiple timestamps or timedeltas.
+
+examples:
+  -
+    - A single timedelta of 2 seconds
+    - |
+      !<tag:weldx.bam.de:weldx/time/time-1.0.0>
+        values: !<tag:weldx.bam.de:weldx/time/timedelta-1.0.0> {value: P0DT0H0M2S}
+  -
+    - A timedelta series from 0 s to 4 s with a 2 s frequency
+    - |
+      !<tag:weldx.bam.de:weldx/time/time-1.0.0>
+        values: !<tag:weldx.bam.de:weldx/time/timedeltaindex-1.0.0>
+            start: !<tag:weldx.bam.de:weldx/time/timedelta-1.0.0> {value: P0DT0H0M0S}
+            end: !<tag:weldx.bam.de:weldx/time/timedelta-1.0.0> {value: P0DT0H0M4S}
+            freq: 2S
+            min: !<tag:weldx.bam.de:weldx/time/timedelta-1.0.0> {value: P0DT0H0M0S}
+            max: !<tag:weldx.bam.de:weldx/time/timedelta-1.0.0> {value: P0DT0H0M4S}
+  -
+    - A single timestamp
+    - |
+      !<tag:weldx.bam.de:weldx/time/time-1.0.0>
+        values: !<tag:weldx.bam.de:weldx/time/timestamp-1.0.0> {value: '2021-08-01T00:00:00'}
+  -
+    - An absolute time index with a 1 s frequency
+    - |
+      !<tag:weldx.bam.de:weldx/time/time-1.0.0>
+        values: !<tag:weldx.bam.de:weldx/time/datetimeindex-1.0.0>
+          start: !<tag:weldx.bam.de:weldx/time/timestamp-1.0.0> {value: '2021-01-01T00:00:00'}
+          end: !<tag:weldx.bam.de:weldx/time/timestamp-1.0.0> {value: '2021-01-01T00:00:02'}
+          freq: S
+          min: !<tag:weldx.bam.de:weldx/time/timestamp-1.0.0> {value: '2021-01-01T00:00:00'}
+          max: !<tag:weldx.bam.de:weldx/time/timestamp-1.0.0> {value: '2021-01-01T00:00:02'}
+        reference_time: !<tag:weldx.bam.de:weldx/time/timestamp-1.0.0> {value: '2021-01-01T00:00:00'}
+
+type: object
+properties:
+  values:
+    type: object
+    oneOf:
+      - tag: "tag:weldx.bam.de:weldx/time/timedelta-1.*"
+      - tag: "tag:weldx.bam.de:weldx/time/timedeltaindex-1.*"
+      - tag: "tag:weldx.bam.de:weldx/time/timestamp-1.*"
+      - tag: "tag:weldx.bam.de:weldx/time/datetimeindex-1.*"
```

```suggestion
      - tag: "tag:weldx.bam.de:weldx/time/timedelta-1.*"
      - tag: "tag:weldx.bam.de:weldx/time/timedeltaindex-1.*"
```
See my comment at the tag. If we change it, reevaluate the examples